src/map/skill.cpp

```diff
@@ -17531,10 +17529,8 @@ bool skill_check_condition_castbegin( map_session_data& sd, uint16 skill_id, uin
 		{
 			if( skill_lv < 3 && ((sd.bl.type == BL_PC && battle_config.pc_cloak_check_type&1)
 			||	(sd.bl.type != BL_PC && battle_config.monster_cloak_check_type&1) )) { //Check for walls.
-				static int dx[DIR_MAX] = { 0, 1, 0, -1, -1,  1, 1, -1};
-				static int dy[DIR_MAX] = {-1, 0, 1,  0, -1, -1, 1,  1};
```

Comment:
Is the change of values intended here? They are different to the ones in dirx/diry.
Rest looks good.

Reply:
I couldn't find a reason for it as the only thing is does it iterate through all directions to see if the skill should fail or not.

I also tested the skill ingame and it still works.